streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -56,7 +56,7 @@ public String name() {
     }
 
     /**
-     * Set the window emit duration in milliseconds of system time
+     * Set the window emit duration in milliseconds of system time.
```

Comment:
"milliseconds since the epoch" is not accurate either, as it is a time interval since the creation of the window.